cmd/fake-lepton/main.go

```diff
@@ -172,22 +173,28 @@ func SendFrames(conn *net.UnixConn, r *cptv.FileReader) error {
     frameSleep := time.Duration(1000/fps) * time.Millisecond
     for {
```

Comment:
this could be `for index := 0; index<=end || end == -1; index++ {`
then can remove index variable